oav.code-workspace

```diff
@@ -0,0 +1,13 @@
+{
+	"folders": [
```

Comment:
I presume this was added by mistake ?

Reply:
no, you can still open using either a folder or the workspace. The workspace contains a relative path to yasway which simplifies testing yasway changes for OAV.